src/common/meta/src/ddl/create_logical_tables/region_request.rs

```diff
@@ -51,13 +55,17 @@ impl CreateLogicalTablesProcedure {
             }
         }
 
-        Ok(RegionRequest {
+        if requests.is_empty() {
```

Comment:
It can't be empty because we've already checked it here.:https://github.com/GreptimeTeam/greptimedb/blob/e88465840d2e11bf65eb4f3907aaf95ee04c0878/src/common/meta/src/ddl/create_logical_tables.rs#L100